src/corehost/common/pal.windows.cpp

```diff
@@ -12,6 +12,9 @@
 #include <ShlObj.h>
 #include <ctime>
 
+#if !defined(RRF_SUBKEY_WOW6432KEY)
+#define RRF_SUBKEY_WOW6432KEY  0x00020000
```

Comment:
This flag wouldn't work on non-win10 operation systems.

/cc @vitek-karas we may want to improve win8-9-2012 coverage for host tests.